src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -21,21 +21,26 @@
 # ARGUMENT DEFINITIONS
 
 allowed_container_types = ['AzureIaasVM']
-allowed_workload_types = ['VM']
+allowed_workload_types = ['VM', 'AzureFileShare']
+allowed_backup_management_types = ['AzureIaasVM', 'AzureStorage']
 
+backup_management_type_help = """Specifiy the backup management type. Defines how Azure Backup manages the backup of entities within the ARM resource. For eg: AzureWorkloads refers to workloads installed within Azure VMs, AzureStorage refers to entities within Storage account. Accepts 'AzureWorkload', 'AzureStorage'. Required only if friendly name is used as Container name."""
```

Comment:
Can you refine your help message for backup_management_type_help?

- Defines -> Define
- Remove  "Accepts 'AzureWorkload', 'AzureStorage'.", because with get_enum_type(), you help message will be appended with "Allowed values: AzureIaasVM, AzureStorage.".